pyproject.toml

```diff
@@ -1,60 +1,192 @@
+# See https://github.com/SciTools/.github/wiki/Linting
+# for SciTools linting guidelines
+
 [tool.black]
 line-length = 79
 target-version = ["py37", "py38", "py39", "py310"]
 include = '\.pyi?$'
 
+[project]
+authors = [
+    {name = "UK Met Office", email = "scitools-iris-dev@googlegroups.com"}
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "Operating System :: OS Independent",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+    "Topic :: Scientific/Engineering :: Visualization",
+]
+description = "Tephigram plotting in Python"
+dynamic = [
+    "dependencies",
+    "optional-dependencies",
+    "readme",
+    "version",
+]
+keywords = [
+    "tephigram",
+    "radiosonde",
+    "meteorology",
+    "visualization",
+]
+license = {text = "BSD-3-Clause"}
+name = "tephi"
+requires-python = ">=3.8"
+
+[project.urls]
+Code = "https://github.com/SciTools/tephi"
+Issues = "issues = https://github.com/SciTools/tephi/issues"
+Binder= "https://mybinder.org/v2/gh/SciTools/tephi/main?filepath=index.ipynb"
+Docs =  "https://tephi.readthedocs.io/en/latest/"
+
 [build-system]
 # Defined by PEP 518
-requires = [
-  "setuptools>=45",
-  "wheel",
-]
+requires = ["setuptools>=45"]
 # Defined by PEP 517
 build-backend = "setuptools.build_meta"
 
+#[tool.pytest.ini_options]
+#minversion = "6.0"
+#addopts = ["-ra", "--showlocals", "--strict-markers", "--strict-config"]
+#xfail_strict = true
+#filterwarnings = ["error"]
+#log_cli_level = "info"
+#testpaths = [
+#  "tephi",
+#]
```

Comment:
I think it would be better to leave this section out entirely as it hasn't been tested yet.

Reply:
https://github.com/SciTools/tephi/pull/172/commits/c0eeb8066621debbf4aa3444c3ddc7ce153179e9